Build.mak

```diff
@@ -17,11 +20,15 @@ $O/libdyaml.a: $(shell find $C/submodules/d-yaml/source -type f -name '*.d') \
 		$(shell find $C/submodules/tinyendian/source -type f -name '*.d')
 	$(call exec, dmd $(DFLAGS) -lib -of$@ $?)
 
+$O/libd-colorize.a: $(shell find $C/submodules/d-colorize/source -type f -name '*.d')
+	$(call exec, dmd $(DFLAGS) -lib -of$@ $?)
+
 ################################################################################
 
 $O/allunittests $O/fastunittest: $O/libvibed.a $O/libdyaml.a
```

Comment:
This is probably missing `$O/libcolorize.a` too.

Reply:
https://github.com/sociomantic-tsunami/neptune/pull/139